helpers/request-helper.js

```diff
@@ -10,5 +10,9 @@ module.exports.getTotpHeaders = (headers = {}) => {
 };
 
 module.exports.getJsonData = (jsonString) => {
+  var isOnlyWhitespace = /^\s+$/.test(jsonString);
```

Comment:
What do you think about this?
```
jsonString = jsonString || '{}'; // only necessary if you can't trust the incoming arg.  I'm guessing that event.body is never null though...

return JSON.parse(jsonString.trim() || '{}');
```

Reply:
Unfortunately, `event.body` _is_ `null` when no body was provided in the request.

Given that, though, I think your two-line version here would work. I'll do that.

Reply:
ok, I appreciate the consideration

Reply:
Done in 64d8eb1c